packages/graphql-tool-utilities/src/augmentations.ts

```diff
@@ -64,6 +56,10 @@ function resolveSchemaPath(this: GraphQLProjectConfig, ignoreMissing = false) {
   return schemaPath;
 }
 
-GraphQLProjectConfig.prototype.resolvePathRelativeToConfig = resolvePathRelativeToConfig;
+// temporary augmentation until `graphql-config` supports this new function
+// see: https://github.com/prisma/graphql-config/pull/113
+GraphQLProjectConfig.prototype.resolvePathRelativeToConfig =
+  GraphQLProjectConfig.prototype.resolveConfigPath;
```

Comment:
I turned this into a real alias instead of a wrapping alias. deleted the wrapper above.